packages/iris-grid/src/ColumnStatistics.tsx

```diff
@@ -170,8 +170,14 @@ class ColumnStatistics extends Component<
       statistics == null &&
       model.isColumnStatisticsAvailable;
     const statisticElements = [];
-    const columnType = column.type.substring(column.type.lastIndexOf('.') + 1);
-    const description = column.description === null ? null : column.description;
+    let columnType = column.type.substring(column.type.lastIndexOf('.') + 1);
+    let description;
+    if (column.description != null && column.description !== undefined) {
```

Comment:
This check is redundant. `!= null` will mean that `!== undefined` is always true. You'd only need to strictly check against `undefined` if you strictly checked against null, e.g. `!== null` instead of just `!= null`.
When checking against both null and undefined, doing a loose check `!= null` is fine.